index.js

```diff
@@ -24,6 +24,8 @@ module.exports = postcss.plugin(
   function fixUrl(options) {
     options = options || {}
     var mode = options.url !== undefined ? options.url : "rebase"
+    var isCustom = typeof mode === "function"
+    var callback = isCustom ? getCustomProcessor(mode) : getUrlProcessor(mode)
```

Comment:
First of all, get `url-processor-function` one time. Earlier it was gotten for each declaration.